src/azure-cli/azure/cli/command_modules/acr/_format.py

```diff
@@ -243,6 +251,29 @@ def _agentpool_format_group(item):
     ])
 
 
+def _connected_acr_format_group(item):
+    return OrderedDict([
+        ('NAME', _get_value(item, 'name')),
+        ('STATUS', _get_value(item, 'status')),
+        ('PARENT', _get_value(item, 'parent')),
+        ('LOGIN NAME', _get_value(item, 'loginName')),
+        ('AUTO UPDATE', _get_value(item, 'autoUpdate')),
+        ('LAST UPDATED', _get_value(item, 'lastUpdated')),
```

Comment:
Does this correspond to the `LastVersionUpdateTime`, `LastSyncTime`, or `LastActivityTime`?

Reply:
ditto

Reply:
the cli experienced didn't specified so I set it to lastActivityTime for now